README.md

```diff
@@ -83,8 +163,9 @@ Get filtered payment methods: (Example to get Free and only Free if *TotalPrice*
     final List<PaymentMethodInfo> = paymentAdapterService.findAvailablePaymentMethods(filter);
 
 ### Creating the Payment 
+
 When the customer selected a payment method that she want to use, then the shop has to call the *createPayment* method with a 
```

Comment:
In order to persist selected payment method in CTP use [createPayment](https://commercetools.github.io/commercetools-payment-integration-java/javadoc/v/current/com/commercetools/payment/service/PaymentAdapterService.html#createPayment-com.commercetools.payment.model.CreatePaymentData-).